### ISSUE TITLE
Console Spam Config Option Planned?

### ISSUE THREAD

**Issue by slidingmike** [gigaherz/ElementsOfPower#13]
It seems that the server console gets spammed only a pretty regular basis. Is there any plan for a config option to stop this spam when world generation happens?

**Comment by gigaherz** [comment]
Yes of course. The message is only there for debugging purposes, since I had some performance issues. I just never removed it yet, but it will be gone before I consider the mod finished ;P

**Comment by slidingmike** [comment]
Thanks! Keep up the good work.

On Sat, Jun 11, 2016 at 6:04 PM, David Quintana notifications@github.com
wrote:

> Yes of course. The message is only there for debugging purposes, since I
> had some performance issues. I just never removed it yet, but it will be
> gone before I consider the mod finished ;P
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/gigaherz/ElementsOfPower/issues/13#issuecomment-225397027,
> or mute the thread
> https://github.com/notifications/unsubscribe/AJfbnsDTTWsnAsTBQ98PT9OVKcnVv83rks5qKzELgaJpZM4Izppv
> .

## 

Michael Wright
ICM Distributing Account Representative
(865) 307-6190
mawright865@gmail.com
